src/test/java/com/epam/asset/tracking/service/UserServiceImplTest.java

```diff
@@ -46,5 +58,46 @@ public void userNotFound() {
     instance.loadUserByUsername("someUserName");
 
   }
+
```

Comment:
Missing tests:
* what happens when the businessProviderService can't find any user?
* what happens when the BusinessProvider in sendEmail() method is null?